### DISCUSSION TITLE
How to recover a sword accidentally given as a gift to a villager.

### DISCUSSION THREAD

**Discussion by CarlosNahuelcoy** [Luke100000/minecraft-comes-alive#705] [Q&A]
Hello, good morning. I was testing the option of giving gifts to villagers, and accidentally, I gave a enchanted diamond sword to a villager archer. Sometimes I see that he equips it, so I assume he keeps it in his inventory as long as I don't give him anything else. Is there any way to retrieve it? Perhaps with a command?

**Comment by Luke100000** [comment]
Right click with the villager editor to open his inventory
